astra-db-java/src/main/java/com/datastax/astra/client/package-info.java

```diff
@@ -4,10 +4,10 @@
  * <p><b>Usage Logic In a nutshell</b></p>
  * <div style="background: #f8f8f8; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .2em;padding:.2em .6em;"><pre style="margin: 0; line-height: 125%"><span style="color: #408080; font-style: italic">// Initialize client</span>
  *DataAPIClient client <span style="color: #666666">=</span> <span style="color: #008000; font-weight: bold">new</span> DataAPIClient<span style="color: #666666">(</span><span style="color: #BA2121">&quot;token&quot;</span><span style="color: #666666">);;</span>
- *<br/><span style="color: #408080; font-style: italic">// Database (crud for collections) work with an assigned namespace</span>
```

Comment:
Self-enclosing tags not allowed by javadoc generator for whatever reason. Go figure.